src/ObjCRuntime/Runtime.cs

```diff
@@ -162,23 +162,17 @@ internal unsafe struct InitializationOptions {
 					return (Flags & InitializationFlags.IsSimulator) == InitializationFlags.IsSimulator;
 				}
 			}
-
-#if NET
-			// Future optimization potential: make the linker change this into a constant.
-			internal bool IsCoreCLR {
-				get {
-					return (Flags & InitializationFlags.IsCoreCLR) == InitializationFlags.IsCoreCLR;
-				}
-			}
-#endif
 		}
 
 		internal static unsafe InitializationOptions* options;
 
 #if NET
 		[BindingImpl (BindingImplOptions.Optimizable)]
 		internal unsafe static bool IsCoreCLR {
-			get { return options->IsCoreCLR; }
+			get {
+				// The linker may turn calls to this property into a constant
+				return (options->Flags & InitializationFlags.IsCoreCLR) == InitializationFlags.IsCoreCLR;
```

Comment:
The line is evil and will make people read it wrong, I'd add a ()

Reply:
it's (mostly) a copy/paste of the existing code, removing a call indirection
I can replace it with `HasFlag` [1] since it's easier/faster to read

[1] https://docs.microsoft.com/en-us/dotnet/api/system.enum.hasflag?view=net-5.0